.github/workflows/build.yml

```diff
@@ -0,0 +1,39 @@
+name: Check the IVOA document
+
+env:
+  doc_name:
```

Comment:
Here the document shortname is missing, ObjVisSAP, to let the build work

Reply:
Done